test/comment/comment_test.dart

```diff
@@ -6,8 +6,9 @@
 import 'dart:async';
 import 'dart:io';
 
-import 'package:test/test.dart';
 import 'package:draw/draw.dart';
+import 'package:draw/src/models/mixins/voteable.dart';
```

Comment:
Add this as an export in `lib/draw.dart` so its contents are visible to anyone importing `package:draw/draw.dart`. Once you do that, you can remove this line here.